tests/case/test_routes.py

```diff
@@ -73,7 +72,7 @@ def test_delete_route(self, client):
     @with_context
     @with_client
     def test_update_case_status_route(self, client):
-        case = CaseHelper._create_case_db()
+        case = CaseHelper._create_case_db(deed_id=24)
```

Comment:
that id is used in several places, should probably be a global/const
http://stackoverflow.com/questions/2682745/creating-constant-in-python